README.md

```diff
@@ -41,6 +41,16 @@ must be applied via [patches](patches). See [below](#patches) for more info on p
 
 ## Updating an External Component to a Newer Version
 
+When updating a component that is used in both Arcade and another target repository, follow these steps:
```

Comment:
> and another target repository

That's not a requirement for this workflow. All that really matters is if Arcade has a dependency.